ffconv/stream_processors.py

```diff
@@ -0,0 +1,240 @@
+"""
+This module contains the actual stream processors, which do the actual
+media conversions.
+"""
+
+from .utils import execute_cmd, log, CalledProcessError
+
+
+class StreamProcessor(object):
+    """
+    Base class for all stream processors, providing the basic interface and
+    common functionality.
+    """
+    media_type = None
+
+    def __init__(self, input, stream, profile):
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Redefining built-in 'input' [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=Pylint%3AW0622)